docs/central-intro.rst

```diff
@@ -32,54 +33,53 @@ Here are some of the major features we support today:
-- :doc:`managed encryption <central-encryption>` makes the process of handling encrypted form data significantly easier and in many cases more secure
+ODK Central Overview
+--------------------
 
+Once you have :ref:`installed Central <central-install>`, you may be wondering where to start.
 
-Please give Central a try and provide your `feedback <https://forum.getodk.org/c/support>`_.
+In Central, everything begins with the :ref:`Project <central-projects>`. Projects organize a Central server into individual little worlds, allowing many separate data collection projects and teams to live on the same server.
 
-.. _central-performance:
+Within a Project can live many :doc:`Forms <central-forms>`. Once you :doc:`author a Form <form-design-intro>` you can upload it to Central, where you can test and preview the Form before publishing it. When it comes time to make changes to a published Form, there is a :ref:`Form Draft <central-forms-updates>` process that helps you update and test the Form safely while important data is still being collected.
 
-Notes On Performance
---------------------
+When a Form is filled out either :ref:`directly on the web in Central <central-submissions-direct>` (using Enketo) or from a mobile client such as :doc:`Collect <collect-intro>`, the finalized data will become a :doc:`Submission <central-submissions>` in Central. Central has features to allow :ref:`review and approval/rejection <central-submissions-review-states>`, :ref:`comments <central-submissions-details>`, and :ref:`edits <central-submissions-editing>` on Submissions. You can also :ref:`bulk download <central-submissions-download>` your data as a file, :ref:`connect to a live feed <central-submissions-odata>` of your data through analysis apps like Power BI or Excel, or use the :ref:`OData API <central-submissions-other-api>` directly to access data.
 
-Central was designed from early on to be stable, predictable, and fast on limited hardware.
+Additionally, Submissions can also create or update :doc:`Entities <central-entities>`. An Entity is a lasting data record: a patient, this tree, that school, and so on. Forms in Central can be configured to :ref:`create <central-entities-registration-forms>` or :ref:`update <central-entities-update>` an Entity record with new information when a Submission is uploaded. This updated Entity data is then sent back out to any data collection clients and can then be :ref:`referenced in future Forms <central-entities-follow-up-forms>`. This allows you to definitively relate repeated encounters with the same Entity together without any homework or guesswork, and it allows you to contextualize and customize the Form filling process using the latest information about a known Entity. In case multiple people try to update an Entity at the same time, there is a :ref:`conflict management system <central-entities-update-conflicts>`. Just like Forms and Submissions, Entities live inside of Projects.
```

Comment:
"an Entity record" -> "an Entity" for consistency. We currently use Entity to mean the representation in ODK.

I would also "Entity data" -> "Entity" but feel less strongly aboutthat one.